### ISSUE TITLE
Add the feature to extend the greeting class for welcoming the new user by user.id in game

### ISSUE THREAD

**Issue by DeKuHigh** [DeKuHigh/UnityLibrary#5]
Add the specific class to inherit the **Traceable.cs** functionality and display the user Data in the game console whenever the player logins for the first time.

**Comment by ErebusXAbyss** [comment]
I would like to work this issue

**Comment by DeKuHigh** [comment]
Yeah sure, just be sure add this feature in the existing **Traceable.cs** class
